docs/modules/ROOT/pages/hazelcast-platform-operator-jet.adoc

```diff
@@ -0,0 +1,339 @@
+* Blob storage and access credentials on one of the following cloud providers: https://aws.amazon.com/s3/[AWS], https://cloud.google.com/storage/[GCP], https://azure.microsoft.com/en-us/services/storage/blobs/[Azure]
+
+== Step 1. Deploy the Jet Pipeline JAR to Cloud Storage
+
+In this step, you’ll build the Jet pipeline JAR from the tutorial’s GitHub repository, and upload it to your cloud provider.
+
+. Clone the Sample Project
+
++
+All the code for this project is available on GitHub. In this step, you'll clone the project and learn how it works.
+
++
+[tabs]
+====
+HTTPS::
++
+--
+```bash
+git clone https://github.com/hazelcast-guides/hazelcast-platform-operator-jet.git
+
+cd hazelcast-platform-operator-jet
+```
+--
+SSH::
++
+--
+```bash
+git clone git@github.com:hazelcast-guides/hazelcast-platform-operator-jet.git
+
+cd hazelcast-platform-operator-jet
+```
+--
+====
+
++
+The sample code for this tutorial is in the link:{github-directory}/tree/master/jet-pipeline/src/main/java/org/example/run[`jet-pipeline/src/main/java/org/example/run/`] directory:
+
+. Build the Jet pipeline JAR
+
++
+[source, shell]
+----
+cd jet-pipeline
+mvn package
+----
+
+. Upload the Jet pipeline JAR to the storage blob/bucket of your cloud provider, replacing the placeholder values:
+
++
+[tabs]
+====
+
+S3::
++
+--
+[source,bash]
+----
+aws s3 cp jet-pipeline/target/jet-pipeline-run-1.0.0.jar s3://<BUCKET_NAME>
+----
+--
+
+GCS::
++
+--
+[source,bash]
+----
+gsutil cp jet-pipeline/target/jet-pipeline-run-1.0.0.jar gs://<BUCKET_NAME>
+----
+--
+
+ABS::
++
+--
+[source,bash]
+----
+az storage blob upload --account-name <ACCOUNT_NAME> --container-name <CONTAINER_NAME> --file jet-pipeline/target/jet-pipeline-run-1.0.0.jar
+----
+--
+
+====
+
+. Create a secret for your cloud storage by running one of the following commands. Remember to replace the placeholder values.
+
++
+[tabs]
+====
+
+AWS::
++
+--
+[source,bash]
+----
+kubectl create secret generic <SECRET-NAME> --from-literal=region=<region> \
+	--from-literal=access-key-id=<access-key-id> \
+	--from-literal=secret-access-key=<secret-access-key>
+----
+--
+
+GCP::
++
+--
+[source,bash]
+----
+kubectl create secret generic <SECRET-NAME> --from-file=google-credentials-path=<service_account_json_file>
+----
+--
+
+Azure::
++
+--
+[source,bash]
+----
+kubectl create secret generic <SECRET-NAME> \
+	--from-literal=storage-account=<storage-account> \
+	--from-literal=storage-key=<storage-key>
+----
+--
+
+====
+
+== Step 2. Start the Hazelcast Cluster
+
+. Run this command to create a Hazelcast cluster with the Jet engine configured.
+
++
+[source, shell]
+----
+cat <<EOF | kubectl apply -f -
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: my-hazelcast
+spec:
+  clusterSize: 3
+  jet:
+    resourceUploadEnabled: true
+EOF
+----
++
+
+NOTE: You need to use Hazelcast 5.3 or further versions to able to submit JetJobs with the Hazelcast Platform Operator.
```

Comment:
```suggestion
NOTE: You need to use Hazelcast version 5.3 or above to submit JetJobs with the Hazelcast Platform Operator.
```

Reply:
I'll update it as you suggested with additional condition. 
It'll be:
NOTE: If you want to specify the Hazelcast version, you need to use Hazelcast 5.3 or above to submit JetJobs with the Hazelcast Platform Operator.